src/main/java/com/codeborne/selenide/proxy/SelenideProxyServer.java

```diff
@@ -67,12 +67,12 @@ public void start() {
     port = proxy.getPort();
   }
 
-  private void addRequestFilter(String name, RequestFilter requestFilter) {
+  public void addRequestFilter(String name, RequestFilter requestFilter) {
```

Comment:
`private` to `public` also required javadoc.
So checkstyle failed.

Reply:
fixed